charts/argo-cd/templates/argocd-application-controller/clusterrolebinding.yaml

```diff
@@ -13,4 +13,11 @@ subjects:
 - kind: ServiceAccount
   name: {{ include "argo-cd.controller.serviceAccountName" . }}
   namespace: {{ include "argo-cd.namespace" . }}
+{{- if .Values.configs.params.applicationsetcontroller.namespaces }}
+{{- range (split "," .Values.configs.params.applicationsetcontroller.namespaces) }}
+- kind: ServiceAccount
+  name: {{ include "argo-cd.controller.serviceAccountName" $root }}
+  namespace: {{ . | trim }}
+{{- end }}
```

Comment:
It seems you didn't check if the Helm template can be rendered properly.

Maybe should be like below ?
```yaml
{{- $namespaces := index .Values.configs.params "applicationsetcontroller.namespaces" -}}
{{- range $namespace := (split "," $namespaces) }}
  {{- if $namespace }}
- kind: ServiceAccount
  name: {{ include "argo-cd.controller.serviceAccountName" $ }}
  namespace: {{ $namespace | trim | quote }}
  {{- end }}
{{- end }}
```

Reply:
I applied it with your help and it works fine. Thank you.